app/services/billable_metrics/aggregations/sum_service.rb

```diff
@@ -54,7 +56,56 @@ def compute_pay_in_advance_aggregation
         return BigDecimal(0) unless event
         return BigDecimal(0) if event.properties.blank?
 
-        BigDecimal(event.properties.fetch(billable_metric.field_name, 0).to_s)
+        unless previous_event
+          value = event.properties.fetch(billable_metric.field_name, 0).to_s
+          update_event_metadata(current_aggregation: value, max_aggregation: value)
+
+          return BigDecimal(value)
+        end
+
+        current_aggregation = BigDecimal(previous_event.metadata['current_aggregation']) +
+          BigDecimal(event.properties.fetch(billable_metric.field_name, 0).to_s)
```

Comment:
Layout/MultilineOperationIndentation: Align the operands of an expression in an assignment spanning multiple lines.